app/views/questions/index.html.erb

```diff
@@ -20,9 +20,10 @@
 
   <div class="row">
     <div class="col-lg-6">
-      <h4>Ask a question here</h4>
+      <h4><%=translation('questions.index.ask_question')%)></h4>
       <% if !current_user %>
-        <p>To ask a question, please <a href="/login?return_to=/questions">log in</a> or <a href="/signup?return_to=/questions">sign up</a> first.</p>
+        <p><%=translation('questions.index.to_ask_question')%>, please <a href="/login?return_to=/questions"><%=translation('questions.index.log_in')%></a><%=translation('questions.index.or')%><a href="/signup?return_to=/questions"><%=translation('questions.index.sign_up')%></a> 
```

Comment:
Thanks for opening this up @1liale 😄 

In the yaml file, the `to_ask_question` contains `please`...   `to_ask_question: To ask a question, please` You can remove it from the line...

```suggestion
        <p><%=translation('questions.index.to_ask_question')%> <a href="/login?return_to=/questions"><%=translation('questions.index.log_in')%></a><%=translation('questions.index.or')%><a href="/signup?return_to=/questions"><%=translation('questions.index.sign_up')%></a> 
```

Reply:
@RuthNjeri I added in the suggested change.